Objects/descrobject.c

```diff
@@ -1501,6 +1502,7 @@ static PyMemberDef property_members[] = {
     {"fset", T_OBJECT, offsetof(propertyobject, prop_set), READONLY},
     {"fdel", T_OBJECT, offsetof(propertyobject, prop_del), READONLY},
     {"__doc__",  T_OBJECT, offsetof(propertyobject, prop_doc), 0},
+    {"name",  T_OBJECT, offsetof(propertyobject, prop_name), READONLY},
```

Comment:
Please remove this line so that *name* is entirely private.

Reply:
Removed, thanks